src/main/resources/XWiki/MapMacro.xml

```diff
@@ -676,6 +692,13 @@ No content.
 {{/code}}
 
 {{map location="Rue des Thermopyles, 75014 Paris" width="400" height="300"}}A beautiful street in Paris{{/map}}
+
+== Map using geographic coordinate ==
+{{code}}
```

Comment:
This comit is supposed to correct the MAP-10 issue, I'll create a new one to replace this code macro by {{{, since there is another call of {{code}} macro on the top